java8/src/main/java/cucumber/runtime/java8/ConstantPoolTypeIntrospector.java

```diff
@@ -42,21 +42,16 @@
         return typeArguments;
     }
 
-    private String getLambdaTypeString(ConstantPool constantPool) {
-        int size = constantPool.getSize();
-        String[] memberRef = null;
-
-        // find last element in constantPool with valid memberRef
-        // - previously always at size-2 index but changed with JDK 1.8.0_60
-        for (int i = size - 1; i > -1; i--) {
```

Comment:
Remove the comment? I'm not sure, the comment is dated since it refers to the but fix, but this code has not been released, should there be a problem, it might be useful to know why there is a loop here.